src/test/scala/InstanceRestartingSpec.scala

```diff
@@ -0,0 +1,136 @@
+import com.tapad.docker.DockerComposeKeys._
+import com.tapad.docker._
+import org.mockito.Matchers._
+import org.mockito.Mockito._
+import org.scalatest.{ BeforeAndAfter, FunSuite, OneInstancePerTest }
+import scala.collection.Iterable
+
+class InstanceRestartingSpec extends FunSuite with BeforeAndAfter with OneInstancePerTest with MockHelpers {
```

Comment:
Hi @edward720. These tests need to be able to execute without docker running locally. It looks like there is probably an additional method that you need to mock out. 

If you run `docker-machine stop` then attempt to run these test you will see what I am talking about.